src/osx/carbon/font.cpp

```diff
@@ -313,11 +318,11 @@ void wxFontRefData::MacFindFont()
         // use font caching
         wxString lookupnameWithSize = wxString::Format( "%s_%u_%d", m_info.m_faceName, traits, m_info.m_pointSize );
 
-        static std::map< std::wstring , wxCFRef< CTFontRef > > fontcache ;
-        static std::map< std::wstring , wxCFRef< CFDictionaryRef > > fontattributescache ;
+        static std::map< std::wstring , OSXFontEntry > fontcache ;
```

Comment:
BTW, I didn't think about it during the original review, but why do we actually need to use `std::wstring` as the map key here? It looks like we could just use `wxString` directly and then we wouldn't even need `ToStdWstring()` call... Of course, this is pretty minor.